apps/studio/src/server/modules/site/__tests__/site.router.test.ts

```diff
@@ -158,6 +160,310 @@ describe("site.router", async () => {
       expect(result).toEqual({ name: site.name })
     })
 
-    it.skip("should throw 403 if user does not have read access to the site", async () => {})
+    it("should throw 403 if user does not have read access to the site", async () => {
+      // Arrange
+      const { site } = await setupSite()
+
+      // Act
+      const result = caller.getSiteName({ siteId: site.id })
+
+      // Assert
+      await expect(result).rejects.toThrowError(
+        new TRPCError({
+          code: "FORBIDDEN",
+          message:
+            "You do not have sufficient permissions to perform this action",
+        }),
+      )
+    })
+  })
+
+  describe("setNotification", () => {
+    beforeEach(async () => {
+      await resetTables("AuditLog", "Site")
```

Comment:
[nit] probably should clear `ResourcePermission` otherwise got inter-test dependencies

Reply:
Ah this is covered by the higher-level `beforeEach`. Let me remove Site here so it's less confusing.